superbench/runner/runner.py

```diff
@@ -359,11 +359,15 @@ def _run_proc(self, benchmark_name, mode, vars):
         logger.info('Runner is going to run %s in %s mode, proc rank %d.', benchmark_name, mode.name, mode.proc_rank)
 
         timeout = self._sb_benchmarks[benchmark_name].timeout
+        env_list = '--env-file sb.env'
+        for k, v in mode.env.items():
+            if v is None:
+                continue
+            env_list += ' -e {k}={v}'.format(k=k, v=str(v).format(proc_rank=mode.proc_rank, proc_num=mode.proc_num))
```

Comment:
why here always need to format `proc_rank=mode.proc_rank, proc_num=mode.proc_num`?

Reply:
to support variables like
```yaml
env:
  CUDA_VISIBLE_DEVICES: {proc_rank}
```